docs/pages/modules/get-started.mdx

```diff
@@ -1,118 +1,42 @@
 ---
-title: Get Started
+title: Get started
 ---
 
 import { CodeBlocksTable } from '~/components/plugins/CodeBlocksTable';
 import { Terminal } from '~/ui/components/Snippet';
 
-## Create a new module
+There are two ways to get started with the Expo Module API: you can either initialize a new module from scratch or add the Expo Module API to an existing module. This guide will walk you through creating a new module from scratch, and the ["Integrating in an existing library" guide](/modules/existing-library) covers the latter case.
```

Comment:
```suggestion
There are two ways to get started with the Expo Module API: you can either initialize a new module from scratch or add the Expo Module API to an existing module. This guide will walk you through creating a new module from scratch, and the ["Integrating in an existing library" guide](./existing-library.mdx) covers the latter case.
```
But let's wait for the confirmation from @Simek

Reply:
I prefer using full links over relative ones, but this is fine both ways. For the extension, as discussed many time already we can just skip it, which should be the default approach. The test links script in question, which earlier required those extensions seems to work now fine, but we also did not run this script for a 1 year plus because it result in 500+ warnings (not always correct anyway, even before MDX v2 and extension change). So TL/DR, link is fine as is.